gravity.sh

```diff
@@ -858,10 +899,30 @@ Options:
   exit 0
 }
 
+repairSelector() {
+  case "$1" in
+    "recover") recover_database=true;;
+    "recreate") recreate_database=true;;
+    *) echo "Usage: pihole -g -r {recover,recreate}
+Attempt to repair gravity database
+
+Available options:
+  pihole -g -r recover    Try to recover a damaged gravity database file.
+                          Pi-hole tries to restore as much as possible
+                          from a corrupted gravity database.
```

Comment:
```suggestion
                          from a corrupted gravity database.
                          If [force] is set, Pi-hole will run the recovery process even when no damage is detected.
```

Reply:
Probably `"force"` here to rule out that someone takes the brackets literally?